src/main/java/it/gov/pagopa/pu/bff/controller/HealthIndicatorController.java

```diff
@@ -0,0 +1,24 @@
+package it.gov.pagopa.pu.bff.controller;
+
+import it.gov.pagopa.pu.bff.model.ServiceStatus;
+import it.gov.pagopa.pu.bff.service.HealthIndicatorService;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Slf4j
+public class HealthIndicatorController {
```

Comment:
we should document this API on the openApi and then implement here the generated interface